.github/workflows/roman_ci.yml

```diff
@@ -91,8 +41,11 @@ jobs:
         DD_GIT_REPOSITORY_URL: ${{ github.repositoryUrl }}
         DD_GIT_COMMIT_SHA: ${{ github.sha }}
         DD_GIT_BRANCH: ${{ github.ref_name }}
-      cache-path: ${{ needs.data.outputs.data_path }}
-      cache-key: data-${{ needs.data.outputs.data_hash }}-${{ needs.data.outputs.crds_context }}
+      cache-path: ${{ needs.data.outputs.path }}
+      cache-key: data-${{ needs.data.outputs.webbpsf_hash }}-${{ needs.data.outputs.crds_context }}
+      cache-restore-keys: |
+        data-${{ needs.data.outputs.webbpsf_hash }}-
+        data-
```

Comment:
This will fall back to any cache that starts with `data-`. I don't think this is necessary (since the `webbpsf_hash` is read from the available hashes) and would open up the possibility that an incorrectly computed webbpsf hash would use other webbpsf data. For example if a hash `abc` was computed but no `data-abc` cache exists and a `data-123` cache exists having `data-` as a restore key would create the new cache with `data-abc-{pmap}` that would contain the `data-123` data.

Reply:
alright, I'll remove that then
```suggestion
```